dali/base/dasds.cpp

```diff
@@ -1369,7 +1382,10 @@ class CDeltaWriter : implements IThreaded
         unsigned deltaTransactionMaxMemMB = config.getPropInt("@deltaTransactionMaxMemMB", defaultDeltaMemMaxMB);
         transactionMaxMem = (memsize_t)deltaTransactionMaxMemMB * 0x100000;
         if (saveThresholdSecs)
+        {
             thresholdDuration = queryOneSecCycles() * saveThresholdSecs;
+            lastSaveTime = get_cycles_now();
```

Comment:
NB: this is an incidental change. Not being set before meant it would save without checking threshold early than intended (not a big deal, but more correct to set time at start)